src/java_bytecode/java_entry_point.cpp

```diff
@@ -359,7 +431,12 @@ bool java_entry_point(
 
     // are we given a main class?
     if(main_class.empty())
-      return false; // silently ignore
+    {
+      res.main_function=symbol;
+      res.error_found=false;
+      res.stop_convert=true;
+      return res; // silently ignore
```

Comment:
In the other silent-ignore case you use `res.main_function=symbolt();` - why is that different here?